docs/pages/GettingStarted.md

```diff
@@ -33,25 +43,51 @@ If your project is not set up for use with the UI5 Tooling yet, follow these ste
 
```

Comment:
Shouldn't we also recommend doing an `npm i -D @ui5/cli` to not rely on the global CLI installation?
Otherwise upgrading the global CLI might break some projects.

Reply:
I mention both now. The global install to have the command available and the local to have the dependency in every project.